plans/main.fmf

```diff
@@ -13,3 +13,16 @@ adjust:
         how: install
         directory: tmp/RPMS/noarch
     when: how == full
+
+    # Disable systemd-resolved to prevent dns failures
+    # See: https://github.com/teemtee/tmt/issues/2063
+  - prepare+:
+      - name: disable-systemd-resolved
+        how: shell
+        script: |
+            systemctl disable systemd-resolved
+            systemctl mask systemd-resolved
+            rm -f /etc/resolv.conf
+            systemctl restart NetworkManager
+            cat /etc/resolv.conf
+    when: trigger == commit
```

Comment:
When I'm running `tmt -c trigger=commit -vv run plan -n '^/plans/features/core$'` on my laptop before pushing to remote, would this `prepare` phase try to stop `systemd-resolved` on my laptop?

Reply:
Hmmm, yes. Probably not a good idea then? Shall we use `initiator` instead?

Reply:
Yeah, might be better. I'm setting `trigger=commit` on purpose, to make sure I don't miss any relevant test case.

Reply:
Hm, but `packit` [does not set](https://artifacts.dev.testing-farm.io/2646b434-927d-40e0-b627-3889c6b04ac8/work-advancedxu86titq/log.txt) `initiator` (yet):

    11:19:40 context: {'arch': ['x86_64'], 'distro': ['fedora-37'], 'trigger': ['commit']}

Any better idea?

Reply:
None. I can drop `trigger=commit` on my end, but then I need a way to run tests locally without messing my system more than with an installation of a couple of packages...

Reply:
Ok, let's go with `initiator` then: f7754455
Packit pull request is already in review: packit/packit-service/pull/2176